src/Models/TaskModel.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require_once 'src/Entities/TaskEntity.php';
+class TaskModel
+{
+    private PDO $db;
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    /**
+     * @return TaskEntity []
+     *
+     */
+    public function getTaskById(int $id): TaskEntity
+    {
+        $query = $this->db->prepare('SELECT `id`, `project_id`, `user_id`, `name`, `description`, `estimate`, `deadline` FROM `tasks` WHERE `id` = :id;');
+        $query->setFetchMode(PDO::FETCH_CLASS, TaskEntity::class);
+        //Line above added to make returned item entity not an array, so can access name
+        $query->execute(['id' => $id]);
+        return $query->fetch();
+    }
+
+    /**
+     * @return TaskEntity []
+     *
+     */
+    public function getTasksByUserID(int $userID, int $projectID): array
```

Comment:
Suggest removing the word Task(s) from these methods - the fact that they are in the task model already gives me that information